### ISSUE TITLE
cloudflare_record object exports incorrectly.

### ISSUE THREAD

**Issue by echoboomer** [cloudflare/cf-terraforming#134]
`cloudflare_record` objects exported by this tool appear to be formatted improperly. For example, this is what exports when using the tool recently:

```hcl
resource "cloudflare_record" "foobar" {
  domain = foobar.com

  name    = "myrecord.foobar.com"
  type    = "A"
  ttl     = "1"
  proxied = "false"

  value = "1.2.3.4"
}
```

If you look at the argument reference for this resource [here](https://www.terraform.io/docs/providers/cloudflare/r/record.html#argument-reference), you can see that `domain` is not a valid argument. It seems like this field should actually be `zone_id` since that one is required. I don't see zone exported at all in its current form.

Thanks for checking this out.

**Comment by chronon** [comment]
I'm running into this an similar issues, which I believe is due to this tool exporting to v1 of the CF provider instead the current version (v2). [Migration examples](https://www.terraform.io/docs/providers/cloudflare/guides/version-2-upgrade.html) are available.

**Comment by orls** [comment]
This was fixed by #128
